debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectionFactory.java

```diff
@@ -5,32 +5,33 @@
  */
 package io.debezium.connector.oracle;
 
-import static io.debezium.connector.oracle.OracleConnectorConfig.CONNECTOR_ADAPTER;
-
 import java.sql.Connection;
-import java.sql.DriverManager;
 import java.sql.SQLException;
 
 import io.debezium.connector.oracle.OracleConnectorConfig.ConnectorAdapter;
 import io.debezium.jdbc.JdbcConfiguration;
+import io.debezium.jdbc.JdbcConnection;
 import io.debezium.jdbc.JdbcConnection.ConnectionFactory;
 
 public class OracleConnectionFactory implements ConnectionFactory {
 
+    private static final String DRIVER_TYPE_THIN = "thin";
+    private static final String CONNECTION_ADAPTER = "connection.adapter";
+    private static final String USE_FETCH_SIZE_WITH_LONG_COLUMN = "useFetchSizeWithLongColumn";
+
     @Override
     public Connection connect(JdbcConfiguration config) throws SQLException {
-        String hostName = config.getHostname();
-        int port = config.getPort();
-        String database = config.getDatabase();
-        String user = config.getUser();
-        String password = config.getPassword();
-
-        final String url = "jdbc:oracle:" + getDriverType(config) + ":@" + hostName + ":" + port + "/" + database;
-
-        return DriverManager.getConnection(url, user, password);
+        final String driverType = getDriverType(config);
+        if (DRIVER_TYPE_THIN.equals(driverType)) {
```

Comment:
I am a little bit confused here - `USE_FETCH_SIZE_WITH_LONG_COLUMN` is set to `true` by default. Why does it need to be overwritten here?

Reply:
Should `USE_FETCH_SIZE_WITH_LONG_COLUMN` be defined in `JdbcConfiguration`?  It's not a known field and so what I saw is that its not being added as part of the default `JdbcConfiguration` object so I had to force add it if the configuration option wasn't being specified by the user's connector setup.

Reply:
You can ignore my comment above, I've removed this configuration option entirely so its a moot point.